packages/rancher-gatekeeper/rancher-gatekeeper.patch

```diff
@@ -992,16 +272,18 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-gatekeeper/charts-original/val
  emitAuditEvents: false
  image:
 -  repository: openpolicyagent/gatekeeper
--  release: v3.1.0
+-  release: v3.1.1
 +  repository: rancher/openpolicyagent-gatekeeper
-+  tag: v3.1.0
++  tag: v3.1.1
```

Comment:
Should this be `tag: 3.1.1`?

Reply:
Nope, they only removed the v from the chart version and not the image or app